packages/components/note/stories/Note.stories.tsx

```diff
@@ -14,18 +15,18 @@ export default {
     className: { control: { disable: true } },
     testId: { control: { disable: true } },
   },
-};
+} as Meta;
 
 interface Args extends NoteProps {
   noteText?: string;
 }
 
-export const basic = ({ noteText, ...args }: Args) => (
+export const basic: Story<any> = ({ noteText, ...args }: Args) => (
```

Comment:
```suggestion
export const basic: Story<any> = ({ noteText, ...args }) => (
```

Reply:
@denkristoffer removing this removes the ability of editing the content of the Note in the story 🤔

Reply:
I guess this should've been 

```suggestion
export const basic: Story<Args> = ({ noteText, ...args }) => (
```

?

Reply:
went around it with using children, should be good too 🙏🏻